File: Level1/수박수박수박수박/수박수박수박수박.java
package Level1.수박수박수박수박;

class Solution {
    public String solution(int n) {
        StringBuilder answer = new StringBuilder();
        for(int i = 1; i <= n; i++)
            answer.append(i%2==0 ? "박" : "수");
        return answer.toString();
    }
}

public class 수박수박수박수박 {
    public static void main(String[] args) {
        Solution solution = new Solution();
        System.out.println(solution.solution(5));
    }
}

File: Level1/수박수박수박수박/수박수박수박수박.kt
package Level1.수박수박수박수박

class Solution {
    fun solution(n: Int): String {
        return (1..n).joinToString("") { i -> if (i % 2 != 0) "수" else "박" }
    }
}

fun main(args: Array<String>) {
    val solution = Solution()
    println(solution.solution(4))
}